vtex-apps/store-image
# Feature/make cms compliant

#### What problem is this solving?

* Make the block `image` compliant with the new CMS
* Bump dependencies versions

#### How to test it?

* [Workspace for the site editor for you to compare if anything changed](https://cmscomponents--storecomponents.myvtex.com/admin/cms/site-editor/about-us)
* [Workspace of the site editor before the changes (nothing should be different)](https://storecomponents.myvtex.com/admin/cms/site-editor/about-us)
* [Workspace of the new CMS, click in the plus button and add a Image block to the page, should have the same schema as the site-editor](https://cmscomponents--storecomponents.myvtex.com/admin/app/new-cms/edit/new)

#### Screenshots or example usage:




#### How does this PR make you feel? [:link:](http://giphy.com/)

![](https://media.giphy.com/media/Ta7lq6ZMBOqzu/giphy.gif)
